config/main.py

```diff
@@ -5151,7 +5179,10 @@ def add_vrf(ctx, vrf_name):
     if (vrf_name == 'mgmt' or vrf_name == 'management'):
         vrf_add_management_vrf(config_db)
     else:
-        config_db.set_entry('VRF', vrf_name, {"NULL": "NULL"})
+        try:
```

Comment:
Should we also use `ValidatedConfigDBConnector` for set_entry?

Reply:
Yes thank you, I added `ValidatedConfigDBConnector` (decorator was present on my DUT, but not in this PR code yet)